docs/2.0/docs/pipelines/installation/addingexistingrepo.mdx

```diff
@@ -0,0 +1,867 @@
+# Bootstrap Pipelines in an Existing Repository
```

Comment:
There's a significant amount of duplicated content between GitHub/GitLab pages.  What do you think of factoring that out?

Reply:
We discussed the issues involved in abstracting away logic into components, requiring work to avoid issues related to backticks and curly braces from interfering with MDX parsing.